ui_src/src/components/tagsPicker/style.scss

```diff
@@ -0,0 +1,135 @@
+.tags-picker-wrapper {
+    width: 100%;
+    align-items: center;
+    align-content: center;
+    overflow-x: hidden;
+    .tags-picker-title {
+        margin-left: 12%;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 16px;
+        line-height: 20px;
+        color: #2e2c34;
+    }
+    .no-new {
+        margin-left: 33px;
+        width: 119px;
+        height: 18px;
+
+        /* Regular Text / 12px / Semibold */
+
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 12px;
+        line-height: 18px;
+        /* identical to box height, or 150% */
+
+        color: #4a495c;
+
+        /* Inside auto layout */
+
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+    }
+    .color-circle {
+        margin-left: 10px;
+        margin-top: 10px;
+        display: inline-block;
+        border-radius: 50%;
+        width: 14px;
+        height: 14px;
+    }
+    .checkmark {
+        position: flex;
+        display: inline-block;
+        margin-left: 10px;
+        margin-top: 10px;
+        transform: rotate(45deg) translate(-50%, -50%);
+        height: 12px;
+        width: 7px;
+        border-bottom: 2px solid rgba(74, 73, 92, 0.8);
+        border-right: 2px solid rgba(74, 73, 92, 0.8);
+    }
+    .no-checkmark {
```

Comment:
Why don't you just set 'display: none'? For what do you need all these properties?

Reply:
so the spacing would be the same

Reply:
Talk to me about it